.github/workflows/sphinx-to-website.yaml

```diff
@@ -0,0 +1,68 @@
+---
+  name: Generate Sphinx Docs and Commit to Website
+  on:
+    schedule:
+      - cron: 0 13 * * 1 # run it once a week
+  jobs:
+      build-sphinx:
+          permissions:
+              contents: write
+  
+          runs-on: ubuntu-latest
+          container: ghcr.io/gem5/ubuntu-24.04_all-dependencies:latest
+          steps:
+              - name: checkout gem5
+                uses: actions/checkout@v4
+                with:
+                    repository: gem5/gem5
+                    path: gem5
+                    ref: stable
+
+              - name: checkout website
+                uses: actions/checkout@v4
+                with:
+                    path: website
+                    ref: stable
+              - name: install Sphinx
+                run: |
+                    apt-get update
+                    apt-get install python3-sphinx -y --fix-missing
+                    sphinx-build --version
+  
+              - name: Cache gem5 # remove caching in final (?)
+                id: cache-gem5
+                uses: actions/cache@v4
+                with:
+                    path: gem5/build/ALL
+                    key: gem5.opt
+              - name: Build gem5
+                # if: steps.cache-gem5.outputs.cache-hit != 'true'
+                # The entire build/ALL folder is cached, so try commenting this
+                # out to see if gem5 will build relatively quickly
```

Comment:
Are you still planning on testing this? I think it's OK if it takes a while to build.

Reply:
I did test this at one point; it looks like the time to build is about the same whether gem5 is built from scratch or if gem5 is built from a cached build/ALL directory. I'll remove the cache steps since it's only needed for testing.